tests/pebble.rs

```diff
@@ -429,7 +469,7 @@ fn verify_cert(
         UnixTime::now(),
         crypto_provider.signature_verification_algorithms.all,
     )
-        .unwrap();
+    .unwrap();
```

Comment:
Nit: move this into the original commit?

Reply:
Done.